distributed/core.py

```diff
@@ -316,7 +342,39 @@ def __init__(
         connection_args=None,
         timeout=None,
         io_loop=None,
+        local_directory=None,
     ):
+        from distributed.nanny import Nanny
+
+        if local_directory is None:
+            local_directory = (
+                dask.config.get("temporary-directory") or tempfile.gettempdir()
+            )
+
+        if "dask-worker-space" not in str(local_directory):
+            local_directory = os.path.join(local_directory, "dask-worker-space")
```

Comment:
I think `dask-temp-space` or `dask-scratch-space` makes more sense now instead of `dask-worker-space`

I view the name of this directory to be internal, but cc @jacobtomlinson in case I'm missing something

Reply:
Went with `dask-scratch-space` for now. https://github.com/dask/distributed/pull/7802/commits/680edd086b1540ffba5857acd8d3b3eb80933dc3

Reply:
I agree that changing the name makes sense given that this is being used by the scheduler too. The only impact I can imagine will be the many `.gitignore` files to update across the community 😆.